app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialRepositoryImpl.java

```diff
@@ -342,6 +342,7 @@ private FieldUiModel transform(@NonNull ProgramStageDataElement stage, DataEleme
                 objectStyle,
                 dataElement.fieldMask(),
                 optionSetConfig,
+                null,
```

Comment:
Here you are setting the autocomplete list to null because we are not using it in the event initial components, given that they are the old ones, right?. Ok, it makes sense

Reply:
exactly